src/components/Routes/Find/Results/SortBy/SortBy.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import './SortBy.scss';
+
+import * as actions from '../../../../../store/actions/index';
+
+class SortBy extends Component {
+  state = {
+    selectValue: this.props.search.sortBy
+  }
+
+  changeSelectValue = (value) => {
+    const { limit, start, query, location, country, radius, jobType, age } = this.props.search;
+
+    this.setState({
```

Comment:
Does swapping the order of the `setState` and `searchSortByChange` execution change anything?